doc: update the steps of customize the default StorageClass (#488)
harvester/docs: docs/rancher/csi-driver.md

./generate_addon_csi.sh <serviceaccount name> <namespace> k3s
```

> ## Customize the default StorageClass

Reviewer: ```suggestion
## Customize the Default StorageClass
```

> Harvester CSI Driver provides the interface to pre-define the default StorageClass. If we do not specify the default StorageClass, the default StorageClass of Harvester CSI Driver will be the same as Harvester cluster.
> 
> To customize the default StorageClass, you can use the parameter `host-storage-class`. Please check with the following example:

Reviewer: ```suggestion
The Harvester CSI driver provides the interface for defining the default StorageClass. If the default StorageClass in unspecified, the Harvester CSI driver and the Harvester cluster use the same default StorageClass. 

You can use the parameter `host-storage-class` to customize the default StorageClass.
```

Reviewer: > Harvester CSI driver will use the default StorageClass of the host Harvester cluster.

Thanks for clarifying. The relationship was not very clear to me. We can change sentence #2 to this: 
"If the default StorageClass in unspecified, the Harvester CSI driver uses the default StorageClass of the host Harvester cluster."

Author: updated! Thanks!

> 1. Create new a StorageClass of the host Harvester cluster
> 
>   ![](/img/v1.3/rancher/create-new-sc.png)

Reviewer: ```suggestion
1. Create a StorageClass for the host Harvester cluster.

    Example:
    ![](/img/v1.3/rancher/create-new-sc.png)
```

> 2. Deploy CSI driver with parameter `host-storage-class`
> 
>   ![](/img/v1.3/rancher/deploy-csi-driver-with-host-storage-class.png)

Reviewer: ```suggestion
1. Deploy the CSI driver with the parameter `host-storage-class`.

    Example:
    ![](/img/v1.3/rancher/deploy-csi-driver-with-host-storage-class.png)
```

> Then, make sure the Harvester CSI driver is ready. You can verify with the following steps.

Reviewer: ```suggestion
1. Verify that the Harvester CSI driver is ready.
```

Reviewer: I converted this to a step. All other steps that follow are substeps of this one.

> 1. Create volume with harvester CSI driver
> 
>   ![](/img/v1.3/rancher/create-volume-with-harvester-csi-driver.png)

Reviewer: ```suggestion
    1. On the **PersistentVolumeClaims** screen, create a PVC. Select **Use a Storage Class to provision a new Persistent Volume** and specify the StorageClass you created.
        
        Example:
        ![](/img/v1.3/rancher/create-volume-with-harvester-csi-driver.png)
```

Reviewer: I changed the description to match what we expect users to do on the UI. In this case, they are supposed to create a PVC, not a volume. Also, there is no mention of the CSI driver on this screen.

> 2. Check the volume status `bound` and its PVC name
>   
>     ![](/img/v1.3/rancher/check-volume-and-pvc-name.png)

Reviewer: ```suggestion
    2. Once the PVC is created, note the name of the provisioned volume and verify that the status is **Bound**.

        Example:
        ![](/img/v1.3/rancher/check-volume-and-pvc-name.png)
```

> 3. Check the PVC name on the host harvester volume page. Then you can see it is created by the StorageClass you configured.
> 
>   ![](/img/v1.3/rancher/check-pvc-name-on-host-harvester-volume-page.png)

Reviewer: ```suggestion
    3. On the **Volumes** screen, verify that the volume was provisioned using the StorageClass that you created.
    
        Example:
        ![](/img/v1.3/rancher/check-pvc-name-on-host-harvester-volume-page.png)
```

Reviewer: I changed the description to match what we expect users to do on the UI. In this case, they must match the volume name and StorageClass name.
